jcommune-model/src/main/java/org/jtalks/jcommune/model/dao/UserDao.java

```diff
@@ -106,4 +106,11 @@
      * @return list of found users
      */
     List<JCUser> findByUsernameOrEmail(String pattern, int count);
+
+    /**
+     * Return a “proxy” of JCUser without hitting the database. If no row found, it will throws an ObjectNotFoundException.
```

Comment:
_May_ return a proxy (without hitting DB). If no row found the object is returned but it throws `ObjectNotFoundException` when properties are first accessed.